pkg/network/ebpf/c/protocols/http2/decoding-defs.h

```diff
@@ -24,7 +24,7 @@
 // The number of frames to parse is determined by HTTP2_MAX_FRAMES_ITERATIONS, resulting in a total defined as:
 // HTTP2_MAX_FRAMES_ITERATIONS * HTTP2_MAX_TAIL_CALLS_FOR_FRAMES_FILTER (for tail call 0-1)
 #define HTTP2_MAX_TAIL_CALLS_FOR_FRAMES_FILTER 2
-#define HTTP2_MAX_FRAMES_TO_FILTER  120
+#define HTTP2_MAX_FRAMES_TO_FILTER  240
```

Comment:
1. wrong value - should be `HTTP2_MAX_FRAMES_ITERATIONS`
2. also, you didn't modify `HTTP2_MAX_TAIL_CALLS_FOR_HEADERS_PARSER` so we're able to process up to 126 frames